src/lib/tuya.ts

```diff
@@ -369,6 +369,15 @@ export const valueConverterBasic = {
     trueFalse: (valueTrue: number | Enum) => {
         return {from: (v: number) => v === valueTrue};
     },
+    legacyOnOff: ( ) => {
```

Comment:
Why not use `onOff` instead?

Reply:
The device sends out this message - Received Zigbee message from 'light', type 'commandDataReport', cluster 'manuSpecificTuya', data '{"dpValues":[{"data":{"data":[0,0,0,0],"type":"Buffer"},"datatype":2,"dp":21}],"seq":51200}' from endpoint 1 with groupID null. Using the onOff is causing this error - Error 2023-07-05 08:26:46Exception while calling fromZigbee converter: Value '0' is not allowed, expected one of true,false}. The implementation for this is not complete yet. Hence pulled the pull request to draft

Reply:
Not needed. reverted that change